frontend/app/components/common/collapsible-section/collapsible-section.directive.ts

```diff
@@ -47,7 +47,7 @@ export class CollapsibleSectionController {
   }
 }
 
-function CollapsibleSection() {
+function CollapsibleSection():any {
```

Comment:
Why not `IDirective` instead of `any`?

Reply:
```
TS2322: Type '{ restrict: string; templateUrl: string; scope: { workPackage: string; }; require: string[]; link...' is not assignable to type 'IDirective'.
  Types of property 'controller' are incompatible.
    Type 'typeof WorkPackageSingleViewController' is not assignable to type 'string | (new (...args: any[]) => IController) | ((...args: any[]) => void | IController) | (stri...'.
      Type 'typeof WorkPackageSingleViewController' is not assignable to type '(string | (new (...args: any[]) => IController) | ((...args: any[]) => void | IController))[]'.
        Property 'push' is missing in type 'typeof WorkPackageSingleViewController'.
```

There is a workaround for some directives by adding `$onInit() { .. }` to the controller, but it does not work for, e.g., link functions.